proto/controlclient.proto

```diff
@@ -54,9 +54,6 @@ message _CreateCacheRequest {
   string cache_name = 1;
 }
 
-message _CreateCacheResponse {
-}
-
```

Comment:
Can we just edit the message in-place here so that `_CreateCacheResponse` comes after `_CreateCacheRequest` and maintains order?

Reply:
yea good call